handler/censusObservations.go

```diff
@@ -128,6 +128,21 @@ func (c *CensusObservations) Get(w http.ResponseWriter, r *http.Request) {
 		}}
 	}
 
+	//check if the dimensions has the area-type (variable) in it, else append
+	//e.g. /population-types/UR/census-observations?dimensions=resident_age_7b&area-type=ltla,E06000008
+	//in this case `dimensions=resident_age_7` is considered as `ltla,dimensions=resident_age_7`
+	addVaraible := true
```

Comment:
added this check if someone passes the area-type as part of dimension parameter too. In that case it won't get append  from area-types, else it will